internal/distro/distro.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2017 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package distro
+
+// Distro-specific settings that can be overridden at link time with e.g.
+// -X github.com/coreos/ignition/internal/distro.mdadmCmd=/opt/bin/mdadm
+var (
+	// Helper programs
+	mdadmCmd   = "/usr/sbin/mdadm"
+	mountCmd   = "/usr/bin/mount"
+	sgdiskCmd  = "/usr/sbin/sgdisk"
+	udevadmCmd = "/usr/bin/udevadm"
```

Comment:
I have a feeling the answer to this question is no, but while we're at this: are there competing implementations for any of these? If there's any silliness like GNU vs BSD netcat where the command line API is different, we might want to comment which version we use.

Reply:
Busybox has a `mount`, but the util-linux one is pretty much the standard. I'm not aware of any competing implementations of the others.